tools/flang1/flang1exe/interf.c

```diff
@@ -515,6 +515,22 @@ init_use_tree(void)
   use_tree = use_tree_end = 0;
 }
 
+void remove_from_use_tree(char* module){
+  USES_LIST* prev = use_tree;
+  for (USES_LIST* n = use_tree; n; n = n->next) {
+
+    if(strcmp(n->use_module->modulename,module)==0){
+        if(n == use_tree){
```

Comment:
Please use alignment that is consistent with the rest of the file.

Reply:
fixed